provenance/models.py

```diff
@@ -1,21 +1,44 @@
+import copy
 from datetime import datetime
-from memoized_property import memoized_property
 
 import sqlalchemy as sa
-import sqlalchemy.orm
 import sqlalchemy.dialects.postgresql as pg
 import sqlalchemy.ext.declarative
+import sqlalchemy.orm
+from memoized_property import memoized_property
 
 Base = sa.ext.declarative.declarative_base()
 
 SHA1_LENGTH = 40
 VALUE_ID_LENGTH = SHA1_LENGTH + 10 # extra 10 for optional file extension info
 
+class Run(Base):
+    __tablename__ = 'runs'
+
+    id = sa.Column(pg.INTEGER, primary_key=True)
```

Comment:
If we make this a UUID, it will make it easier to move artifacts from one database to another.